documentation/man/features/mail.rst

```diff
@@ -12,6 +12,7 @@ SYNOPSIS
 | *kw mail* (-l | \--list)
 | *kw mail* \--verify [\--local | \--global]
 | *kw mail* \--template[=<template>] [-n | \--no-interactive] [\--local | \--global] [-f | \--force] [(<config> <value>)...]
+| *kw mail* (-V | \--verbose)
```

Comment:
In this case, let's drop the -V option

Reply:
What do you mean when you say 'drop the -V option'? Are you suggesting removing the short option (-V) for verbose, leaving only the long option (--verbose)?

Reply:
just remove the short option `-V` for the mail feature.